migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/MisoServiceManager.java

```diff
@@ -414,12 +422,14 @@ public void setDefaultProjectDao() {
     dao.setNamingScheme(getNameableNamingScheme(Project.class));
     dao.setWatcherDAO(watcherDao);
     dao.setDataObjectFactory(dataObjectFactory);
+    dao.setReferenceGenomeDao(referenceGenomeDao);
     setProjectDao(dao);
   }
 
   private void updateProjectDaoDependencies() {
     if (studyDao != null) studyDao.setProjectDAO(projectDao);
     if (sampleNumberPerProjectService != null) sampleNumberPerProjectService.setSqlProjectDAO(projectDao);
+    if (referenceGenomeService != null) referenceGenomeService.setReferenceGenomeDao(referenceGenomeDao);
```

Comment:
This method is for updating other objects' dependencies on the Project DAO, so this doesn't belong here. There should be an `updateReferenceGenomeDaoDependencies` method which sets the referenceGenomeDao for projectDao and referenceGenomeService